src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    const queryToLower = query.toLowerCase();
```

Comment:
```suggestion
    const queryToLower = query.toLowerCase().trim();
```

(let's also rename it to smth like `formattedQuery`)

<img width="634" alt="image" src="https://user-images.githubusercontent.com/55802630/184636221-2def5ee5-6023-46c9-a24d-af6db7d7ba56.png">